h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -510,7 +510,16 @@ public static final TwoDimTable makeTwoDimTable(String tableHeader, String sort_
     String[] rowHeaders = new String[length];
     for (int i = 0; i < length; i++) rowHeaders[i] = "" + i;
 
-    if ("mean_per_class_error".equals(sort_metric)){ //Multinomial
+    if (sort_metric == null && length == 0) {
```

Comment:
What case can cause this? I thought at least one model is built, i.e., AutoML waits to finish the job it started.